presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -303,6 +303,10 @@ Property Name                                Description
                                              set this to the AWS region-specific endpoint
                                              (e.g., ``http[s]://<bucket>.s3-<AWS-region>.amazonaws.com``).
 
+``hive.s3.storage-class``                    The S3 storage class to use when writing the data. Currently only
+                                             ``Standard`` and ``IntelligentTiering`` storage classes are supported.
+                                             Default storage class is ``Standard``
```

Comment:
Update the possible values here. They should be spelled "STANDARD" and "INTELLIGENT_TIERING".

Reply:
@findepi Fixed. Forgot to push the changes earlier.